### PR TITLE
Re-enable tests

### PR DESCRIPTION
**Pull request by AaronFriel**
We should continue to test the Aiven provider to maintain a quality bar, and monitor undeleted resources or excess spend on the account going forward.

### CODE REVIEW

**Review by t0yv0 — APPROVED** [review]

**Comment by t0yv0** [comment]
@AaronFriel ping - any plans to move this forward or it moves somewhere on the board? Thank you.

**Comment by AaronFriel** [comment]
@t0yv0 thanks for the ping. It looks like tests failed, rerunning then will diagnose.

File: examples/examples_nodejs_test.go [hunk 1]
```diff
@@ -21,16 +20,9 @@ func TestAccService(t *testing.T) {
 }
 
 func TestAccPgService(t *testing.T) {
-	t.Skip() // due to https://github.com/pulumi/pulumi-aiven/issues/235
 	test := getJSBaseOptions(t).
 		With(integration.ProgramTestOptions{
 			Dir: path.Join(getCwd(t), "pg-service"),
-			EditDirs: []integration.EditDir{
```
Comment (t0yv0) [inline examples/examples_nodejs_test.go:28]: Was this EditDirs check causing trouble that we intentionally remove it?

Comment (AaronFriel) [reply examples/examples_nodejs_test.go:28]: The removed test relied on a removed `Service` Resource. That was removed in our v5.

**Pull request closed without merging**